.github/workflows/ElastixGitHubActions.yml

```diff
@@ -18,20 +18,20 @@ jobs:
           - os: ubuntu-18.04
             c-compiler: "gcc"
             cxx-compiler: "g++"
-            itk-git-tag: "v5.1.1"
```

Comment:
Should this ITK upgrade not be done in a separate commit or PR? Or is it related to clang-format?

Reply:
Yes, this was already done in commit 98aaaf7e27c7dacd271db5a2cc823b938a154392, so I don't get why it still shows up here.
But no it's not related to clang-format.

Reply:
git happens 😸